build-advisories/src/Roave/SecurityAdvisories/Version.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Roave\SecurityAdvisories;
+
+/**
+ * A simple version, such as 1.0 or 1.0.0.0 or 2.0.1.3.2
+ */
+final class Version
+{
+    const VALIDITY_MATCHER = '^(\d+.)*\d+$';
+
+    /**
+     * @var string
+     */
+    private $versionNumbers;
+
+    /**
+     * @param int[] $versionNumbers
+     */
+    private function __construct(array $versionNumbers)
+    {
+        $this->versionNumbers = $versionNumbers;
+    }
+
+    /**
+     * @param string $version
+     *
+     * @return self
+     *
+     * @throws \InvalidArgumentException
+     */
+    public static function fromString($version)
+    {
+        if (! preg_match('/' . self::VALIDITY_MATCHER . '/', $version)) {
```

Comment:
why adding the regex delimiter here through concatenation instead of putting them in the constant directly ?